x-pack/filebeat/input/http_endpoint/handler.go

```diff
@@ -59,14 +61,30 @@ func (h *httpHandler) apiResponse(w http.ResponseWriter, r *http.Request) {
 		headers = getIncludedHeaders(r, h.includeHeaders)
 	}
 
+	var responseCode int
+	var responseBody string
```

Comment:
```suggestion
	var (
		status int
		body   string
	)
```

Reply:
This rename generates conflicts since `status` and `body` are defined above and `body` is an `io.ReadCloser`. I've changed the structure but kept the naming, please let me know if it doesn't work for you.

Reply:
Yeah, that's fine. Thanks.